README.md

```diff
@@ -8,7 +8,7 @@
 </h1>
 <h4 align="center">A highly customizable snake clone made in Rust with the <a href="https://github.com/bevyengine/bevy">Bevy engine</a>, named after the Japanese word for snake, <ruby>蛇<rp>(</rp><rt>へび</rt><rp>)</rp></ruby>.</h4>
 
-## Configuration
+## Configuration <a name="configuration"></a>
```

Comment:
As far as I know, this isn't needed for GFM, renderers already generate links to headings in `kebab-case`.

Reply:
I had no idea, thanks for letting me know :)